source/reporting/index.html.md.erb

```diff
@@ -332,6 +332,10 @@ For example:
 }
 ```
 
+### Get notified of payment state changes
+
+You can create webhooks to notify you when payments reaches certain states. Read our [guidance about using webhooks to getg automatic payment updates](/webhooks).
```

Comment:
```suggestion
You can create webhooks to notify you when payments reaches certain states. Read our [guidance about using webhooks to get automatic payment updates](/webhooks).
```

Reply:
Not as sure but would it be "when payments reach certain states"? I've no idea with the number of plurals in there!

Reply:
It should! Good catch.